invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -115,12 +116,22 @@ export class RDMDepositForm extends Component {
           />
         </Overridable>
 
-        <Overridable id="InvenioAppRdm.Deposit.CommunityHeader.container">
-          <CommunityHeader imagePlaceholderLink="/static/images/square-placeholder.png" />
-        </Overridable>
+        {!permissions?.can_publish_restricted && (
+          <Overridable id="InvenioAppRdm.Deposit.CommunityHeader.container">
+            <CommunityHeader imagePlaceholderLink="/static/images/square-placeholder.png" />
+          </Overridable>
+        )}
         <Container id="rdm-deposit-form" className="rel-mt-1">
           <Grid className="mt-25">
             <Grid.Column mobile={16} tablet={16} computer={11}>
+              {/* Active when config "RDM_ENSURE_RECORD_COMMUNITY_EXISTS" = True */}
+              {permissions?.can_publish_restricted && (
+                <Overridable id="InvenioAppRdm.Deposit.CommunityHeaderRestricted.container">
+                  <AccordionField active label={i18next.t("Community")}>
+                    <CommunityHeader imagePlaceholderLink="/static/images/square-placeholder.png" />
+                  </AccordionField>
+                </Overridable>
+              )}
```

Comment:
The objective of this modification is to integrate the community header into the overall flow since numerous users have reported that it's easy to overlook. Given that this design choice hasn't been previously discussed, I welcome any suggestions or alternative approaches to accomplish this aim.